client/my-sites/upgrades/domain-management/email/add-google-apps-card.jsx

```diff
@@ -32,9 +32,9 @@ const AddGoogleAppsCard = React.createClass( {
 		let price = gapps && gapps.cost_display;
 
 		// Gapps price is stored annually but we'd like to show a monthly price
-		price = price.replace( /(\d+\.?\d+)/, ( val ) => {
+		price = price.replace( /(\d+[.,]?\d+)/, ( val ) => {
```

Comment:
Anyone know some locales that have a different price formatting that will fail here?